ironmon_tracker/screens/TrackerScreen.lua

```diff
@@ -438,12 +438,35 @@ end
 
 -- Define each Carousel Item, must will have blank data that will be populated later with contextual data
 function TrackerScreen.buildCarousel()
+	-- Helper functions
+	local function isCarouselDisabled()
+		return Options["Disable mainscreen carousel"] and Battle.isViewingOwn
+	end
+	local function overrideEarlyRouteInfo()
```

Comment:
the naming of this isn't intuitive, at a first glance i took this to mean "returns true if we want to override showing the early route info and show something else" but instead it seems to be "returns true if we want to override what is currently there with the early route info"?
Perhaps something like "showEarlyRouteInfo" or "showEarlyRouteEncounters" could be a more intuitive name in that case